src/components/Button/Button.style.css

```diff
@@ -0,0 +1,62 @@
+.button {
+  width: 150px;
+  height: 44px;
+  text-align: center;
+  font-weight: bold;
+  font-size: 18px;
+  border-radius: 4px;
+  border-width: thin;
+  background-color: #f2f1e7;
+  font-family: "Orqquidea-Sans_(DEMO)";
+  flex-shrink: 1;
+}
+.button__red {
```

Comment:
Should be `--` instead of `__` because you are 'modifying' the style of the element

Reply:
Okey, and when do we use __?

Reply:
You use __ when you are selecting a child/grandchild of a parent class, for example `.footer__first-section`, `.profile__avatar`, `.profile__avatar__image`. You can have a detailed explanation here http://getbem.com/

Reply:
and `--` would be smth like `.profile__avatar--rounded`, `.button--red`, `.header--hidden`